control-plane/api-gateway/common/translation.go

```diff
@@ -113,6 +113,10 @@ func (t ResourceTranslator) toAPIGatewayListener(gateway gwv1beta1.Gateway, list
 		}
 	}
 
+	//grab policy if it exists
```

Comment:
I think the linter generally prefers comments that are sentence cased and end in a period.